src/hearings/store/reducers/hearing-actuals.reducer.ts

```diff
@@ -31,6 +31,13 @@ export function hearingActualsReducer(
         }
       };
     }
+    case fromActions.SUBMIT_HEARING_ACTUALS_SUCCESS: {
+      return {
+        ...currentState,
+        hearingActualsMainModel: null,
+        lastError: null
+      };
```

Comment:
```suggestion
      return {
        ...initialHearingActualsState
      };
```

Reply:
Thank you @rayliangatos , I have updated the code as per your suggestion.